proto/quicksilver/interchainstaking/v1/interchainstaking.proto

```diff
@@ -49,6 +49,13 @@ message Zone {
   bool deposits_enabled = 26;
   bool return_to_sender = 27;
   bool is_118 = 28;
+  SubzoneInfo subzoneInfo = 29;
+}
+
+message SubzoneInfo {
+  string authority = 1;
+  string base_chainID = 2;
+  string chainID = 3;
 }
```

Comment:
needed to add `chainID` here so that we pass this value into the `RegisterZoneProposal` as a unique identifier that is not generated from the `connectionID`